### DISCUSSION TITLE
[Announcement 📢]: Bugs list 📃

### DISCUSSION THREAD

**Discussion by nhatminh1401** [Strypper/mauisland#121] [Bugs list]
I see the app has been improved. 
But there are still some unexpected errors

1.When you on Galley -> Built-in-> TabbedPage -> Detail -> I can't back to Menu Galley
![2023-03-13-15-26-10](https://user-images.githubusercontent.com/82369529/224758368-3b7e81f7-4643-4684-9d23-29e32c0b4ab0.gif)
2. When you on Galley -> Built-in-> Check Box-> More detail to open the new page -> It can't do it and got out of the program
![2023-03-13 15-23-42](https://user-images.githubusercontent.com/82369529/224758356-c2c4ebf6-cd1e-41a9-914b-720a0bf96fa6.gif)
3. When you on Galley -> Syncfusion -> SfRadialGauge, SfBadgeView -> More detail to open the new page-> It can't do it and got out of the program
![2023-03-13 15-21-50](https://user-images.githubusercontent.com/82369529/224758335-f21d6778-85d3-41ff-aad7-2d002dc2d0f4.gif)
![2023-03-13 15-28-57](https://user-images.githubusercontent.com/82369529/224758359-28dba51b-5bb1-45e7-b3a5-ef156124939f.gif)

**Comment by Strypper** [comment]
Thanks we did replicate all of the bugs, fixing it

**Comment by Strypper** [comment]
I have extract the content and create these following issues:

1. [TabbedPage](https://github.com/Strypper/mauisland/issues/125)
2. [SfRadialGaugePage](https://github.com/Strypper/mauisland/issues/124)
3. [CheckboxPage](https://github.com/Strypper/mauisland/issues/123)
